.ci/pytorch/common_utils.sh

```diff
@@ -153,6 +153,14 @@ function install_filelock() {
   pip_install filelock
 }
 
+function install_matplotlib() {
+  pip_install matplotlib
```

Comment:
If you are looking to reduce the chance of flakiness coming from pip installation, I think it's better to add these two dependencies into https://github.com/pytorch/pytorch/blob/master/.ci/docker/requirements-ci.txt instead.  They will become part of the Docker image used for this the perf job.  Then `install_*` can be removed from the CI.

Reply:
Good point. Is there a specific image config I should add to? These libs are not needed for every single test, so I think we should avoid exploding the common image?

Reply:
You can tweak this if you want to.  The `requirements-ci.txt` file is for all images and adding 1 or 2 dependencies there is fine.  

FYI, The one you need is https://github.com/pytorch/pytorch/blob/master/.github/workflows/inductor-perf-test-nightly.yml#L24 (`pytorch-linux-bionic-cuda11.7-cudnn8-py3-gcc7`).  And you can tweak it logic in https://github.com/pytorch/pytorch/blob/master/.ci/docker/build.sh#L104-L116, i.e. having a param like `WITH_DASHBOARD_DEP=yes` then tweak the shared Dockerfile accordingly https://github.com/pytorch/pytorch/blob/master/.ci/docker/ubuntu-cuda/Dockerfile#L124.  But it looks rather complicated IMO.  A slightly different way is to create a brand new Docker image, but  the same principal applies.

Reply:
OK, sounds like not worth the effort for small libs. But on a related note, the other request for having an image with preinstalled benchmarks probably needs to have its own image.

Reply:
IMO, it is worth doing even for small libs. The big benefit is that a version upgrade from upstream doesn't silently break our CI.